modules/oe_content_publication/config/post_updates/00003_create_fields/field.field.node.oe_publication.oe_publication_thumbnail.yml

```diff
@@ -0,0 +1,34 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_publication_thumbnail
+    - media.type.av_portal_photo
+    - media.type.image
+    - node.type.oe_publication
+  module:
+    - composite_reference
+third_party_settings:
+  composite_reference:
+    composite: false
+id: node.oe_publication.oe_publication_thumbnail
+field_name: oe_publication_thumbnail
+entity_type: node
+bundle: oe_publication
+label: Thumbnail
+description: ''
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  handler: 'default:media'
+  handler_settings:
+    target_bundles:
+      av_portal_photo: av_portal_photo
+      image: image
+    sort:
+      field: _none
+    auto_create: false
+    auto_create_bundle: av_portal_photo
```

Comment:
Do we need autocreate?

Reply:
The **av_portal_photo** bundle is removed as an **auto_create_bundle** setting.